client/src/components/bundle/Bundle.tsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from "react";
+import styled from "styled-components";
+import Collapsed from "./Collapsed";
+import Expanded from "./Expanded";
+
+
+
+function Bundle() {
```

Comment:
Exported functions in TypeScript should be annotated with the return type of the function. Like so:


```ts
function Bundle(): JSX.Element { /*...*/ }
```

Or personally, I prefer const lambda functions:

```ts
const Bundle = (): JSX.Element => { /*...*/ }
```

Reply:
Resolved